worklets/Overview.bs

```diff
@@ -262,49 +265,83 @@ is empty; it is populated when the user agent chooses to create its {{WorkletGlo
 A {{Worklet}} has a list of the <dfn>worklet's resolved module URLs</dfn>. Initially this list is
 empty; it is populated when module scripts resolved.
 
+A {{Worklet}} has a <dfn>fetched modules map</dfn>. This is a map of <a>fetch</a> requests to values
```

Comment:
You want this to be a map of request URLs -> responses. Because "fetch a module script tree" will give you a new request every time, so you can't use the requests as cache keys. This also allows you to eliminate the "worklet's list of resolved module URLs"